pipelines/skylab/optimus/Optimus.changelog.md

```diff
@@ -1,3 +1,10 @@
+# 4.5.0
```

Comment:
+1

Reply:
If the outputs are scientifically identical and there are no input /output changes to the pipeline, then it'd be 4.0.1

Reply:
updated to 4.0.1

Reply:
I would think this would be a minor release since it's such a big improvement, but i'm not going to insist

Reply:
That's a really good note - thanks Jess! We're still refining how we determine our versions so this would be a good quick team discussion so we can update the guidelines if we want to. Apologies for stepping in, I was just speaking from interpreting the guidelines as they've been written.